rig-mongodb/src/lib.rs

```diff
@@ -27,10 +27,9 @@ struct SearchIndex {
 impl SearchIndex {
     async fn get_search_index(
         collection: mongodb::Collection<bson::Document>,
-        index_name: &str,
     ) -> Result<SearchIndex, VectorStoreError> {
         collection
-            .list_search_indexes(index_name, None, None)
+            .list_search_indexes()
```

Comment:
Need to add `.name(...)` to builder, otherwise this will fail if there are multiple indices:
```rust
collection
    .list_search_indexes()
    .name(index_name)
    .await
    ...
```

Reply:
See [ListSearchIndexes](https://docs.rs/mongodb/latest/mongodb/action/struct.ListSearchIndexes.html)